packages/support/src/settings/_variables.layout.scss

```diff
@@ -64,3 +49,15 @@ $grid-form-gutter-width: $spacer-2 !default;
 
 // Z-index
 $z-dialog: 1000 !default;
+
+// Site variables
+$site-margins: 3rem !default;
```

Comment:
These variables are *mostly* unused in our repo, however I am keeping them because some are used i.e. `$site-max-width`

Reply:
Couldn't we remove the unused variables? What's the reasoning for keeping them? Maybe someone else is using them at another design system level?

Reply:
Yes, in case someone who is using the CMS DS is using them. I would be ok with removing them normally, but some of these variables are being used by several of our components